SQL/tgstation_schema_prefixed.sql

```diff
@@ -148,6 +148,7 @@ CREATE TABLE `SS13_death` (
   `toxloss` smallint(5) unsigned NOT NULL,
   `cloneloss` smallint(5) unsigned NOT NULL,
   `staminaloss` smallint(5) unsigned NOT NULL,
+  `last_words` varchar(96) DEFAULT NULL
```

Comment:
Some quick maths
The longest message, 104 characters, `i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i i...`
will become 257 characters `i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i... i......` in a freak occurrence of byond RNG popping a 1 in 1x10^51 chance.

From a bunch of recent rounds, average message length is ~20 characters with the longest message each round usually being ~180 characters.

Given this I think `varchar(255)` would be best if you want to catch (almost) everything since it's close to the theorized max of 257 and a good rounding post for databases (max length before data size is length+2 bytes)

However it's up to @nfreader if he thinks it's worth trying to record messages like this since any this size will be bogus text.